config/config.go

```diff
@@ -101,6 +107,7 @@ type configData struct {
 	LogConfig           []logging.LoggerPatternConfig `json:"log,omitempty"`
 	Revision            string                        `json:"revision,omitempty"`
 	MaintenanceConfig   *MaintenanceConfig            `json:"maintenance,omitempty"`
+	PackagePath         string                        `json:"package_path,omitempty"`
```

Comment:
We were missing some code to marshal and unmarshal the `PackagePath` field. I needed that functionality for `TestMachineState`, so I just added it.